src/app/routes/api/v3/user/user.routes.ts

```diff
@@ -42,4 +43,19 @@ UserRouter.post('/contact/otp/generate', UserController.handleContactSendOtp)
  */
 UserRouter.post('/contact/otp/verify', UserController.handleContactVerifyOtp)
 
+/**
+ * Verify the contact verification one-time password (OTP) for the user as part
+ * of the contact verification process
+ * @route POST /user/whats-new/last-seen
+ * @param body.latestLastSeenFeatureUpdateDate the latest seen feature update date to update user with
+ * @returns 200 when user last seen feature update updates sucessfully
+ * @returns 401 if user is not currently logged in
+ * @returns 422 when userId does not exist in the database
+ * @returns 500 when database errors occurs
+ */
+UserRouter.post(
+  '/whats-new/last-seen',
+  UserController.handleUpdateUserLastSeenFeatureUpdateDate,
+)
```

Comment:
wondering if we should rename this route to `/flags/last-seen-feature` instead? since it's not really operating on the any `what's new` collection anymore. We can even add a UserFlagsRouter subrouter to contain it if we want to.

Reply:
I am not really sure what the purpose of a subrouter is in this context. Could you please elaborate as to why we would need to add a subrouter?